rules/mastg-android-sensitive-data-in-notifications.yml

```
@@ -0,0 +1,11 @@
+rules:
+  - id: mastg-android-sensitive-data-in-notifications
+    languages:
+      - java
+    severity: WARNING
+    metadata:
+      summary: This rule looks for notifications that may contain sensitive data.
+    message: "[MASVS-PLATFORM-3] Ensure that notifications do not contain sensitive information"
+    pattern-either:
+      - pattern: $X.setContentTitle(...)
+      - pattern: $X.setContentText(...)
```

Adding `minSdkVersion`. FYI we'll stop using semgrep soon so don't pay too much attention to the `message`s and other metadata.

```suggestion
rules:
  - id: mastg-android-sensitive-data-in-notifications-manifest
    languages:
      - xml
    severity: WARNING
    metadata:
      summary: This rule inspects AndroidManifest.xml for notification post permission. Notification may contain sensitive data.
    message: "[MASVS-PLATFORM-3] Ensure that notifications do not contain sensitive information"
    pattern: android:name="android.permission.POST_NOTIFICATIONS"
  - id: mastg-android-minsdkversion
    languages:
      - xml
    severity: INFO
    metadata:
      summary: This rule inspects AndroidManifest.xml to obtain the minSdkVersion value.
    message: "[MASVS-PLATFORM-3] Ensure that the minSdkVersion is evaluated for notification exposure on older Android versions"
    pattern: android:minSdkVersion="$X"
```